pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "alune"
-version = "0.0.1"
+version = "0.0.2"
```

Comment:
> Bumped version - I'm honestly not sure how we want to bump versions. Do we just follow semantic? I'm happy to bump minor instead of patch.

Personally I like semantic since there really isn't any "perfect" version system but it's got the least concessions IMO, but I'm also okay with something else. Heck, even just an incrementing number is fine, as long as we're consistent about it! 😛 

I would say this is probably a minor version bump tbh, since this is one of the "critical" features we talked about before we'd consider it "good enough", but not feeling nitpicky about that at all! 😅

Reply:
I completely agree with you, I'm not even sure why I bumped patch for this in the first place.